### PR TITLE
renamed metal construction preset

### PR DESCRIPTION
**Pull request by animesh-007**
Renamed Metal Construction preset to Metal Working #7549 
Screen Shot 2020-05-05 at 4 48 23 PM

### CODE REVIEW

File: data/presets/presets/craft/metal_working.json [hunk 1]
```diff
@@ -7,5 +7,5 @@
     "tags": {
         "craft": "metal_construction"
     },
-    "name": "Metal Construction"
+    "name": "Metalworking"
```
Comment (1ec5) [inline data/presets/presets/craft/metal_working.json]: Let’s move this file back to where it was. I don’t think anything depends on the file names being consistent with the tags, but it’ll be easier to find the preset at the old name.

File: data/presets.yaml [hunk 2]
```diff
@@ -4467,10 +4467,10 @@ en:
       craft/locksmith:
         # craft=locksmith
         name: Locksmith
-      craft/metal_construction:
+      craft/metal_working:
```
Comment (1ec5) [inline data/presets.yaml]:
```suggestion
      craft/metal_construction:
```

File: data/presets/presets.json [hunk 3]
```diff
@@ -410,7 +410,7 @@
     "craft/insulator": {"icon": "temaki-tools", "geometry": ["point", "area"], "tags": {"craft": "insulation"}, "name": "Insulator"},
     "craft/joiner": {"icon": "temaki-tools", "geometry": ["point", "area"], "tags": {"craft": "joiner"}, "terms": ["furniture"], "name": "Joiner"},
     "craft/key_cutter": {"icon": "fas-key", "geometry": ["point", "area"], "tags": {"craft": "key_cutter"}, "name": "Key Cutter"},
-    "craft/metal_construction": {"icon": "temaki-tools", "geometry": ["point", "area"], "tags": {"craft": "metal_construction"}, "name": "Metal Construction"},
+    "craft/metal_working": {"icon": "temaki-tools", "geometry": ["point", "area"], "tags": {"craft": "metal_construction"}, "name": "Metalworking"},
```
Comment (1ec5) [inline data/presets/presets.json]:
```suggestion
    "craft/metal_construction": {"icon": "temaki-tools", "geometry": ["point", "area"], "tags": {"craft": "metal_construction"}, "name": "Metalworking"},
```

**Review by 1ec5 — COMMENTED** [review]
Thanks for making this change! There are a few things left to clean up so that the raw tag remains `craft=metal_construction`. Once you make these changes, remember to rebuild iD (`npm install && npm run all`) to update the remaining files.

**Comment by animesh-007** [comment]
@1ec5 Thanks for suggestions. I have made those changes.

File: data/presets/presets.json (hunk 3 above)
Comment (1ec5) [reply data/presets/presets.json]: I implemented this change and force-pushed to your branch.

File: dist/locales/en.json [hunk 4]
```diff
@@ -10708,10 +10708,6 @@
                 "name": "OpenStreetMap Galicia Twitter",
                 "description": "Follow us on Twitter at {url}"
             },
-            "Galicia-Xeoinquedos-Twitter": {
```
Comment (1ec5) [inline dist/locales/en.json]: @quincylvania I ended up introducing a lot of diffs related to the community index and NSI after rebasing and building; might want to double-check that I’m not blowing anything away due to something missing in my configuration.

**Review by 1ec5 — APPROVED** [review]

File: dist/locales/en.json (hunk 4 above)
Comment (quincylvania) [reply dist/locales/en.json]: Hmm you probably need to run `npm install` to sync up your versions of these dependencies before building again.

**Comment by quincylvania** [comment]
What do you all think about the name "Metalworker" instead? Lots of the `craft` presets are like "Plumber", "Roofer", "Painter", "Photographer"…

File: dist/locales/en.json (hunk 4 above)
Comment (1ec5) [reply dist/locales/en.json]: Thanks, that worked.

**Comment by 1ec5** [comment]
That seems reasonable to me.

**Comment by quincylvania** [comment]
@animesh-007 @1ec5 Thanks!!

**Pull request merged**